bundle/manifests/components.platform.opendatahub.io_workbenches.yaml

```diff
@@ -74,6 +74,13 @@ spec:
                       type: object
                     type: array
                 type: object
+              workbenchNamespace:
+                default: rhods-notebooks
+                description: Namespace for workbenches to be installed, configurable
+                  only once when workbenches are enabled, defaults to "rhods-notebooks"
+                maxLength: 63
```

Comment:
@harshad16 Fairly certain this length won't work for creating OpenShift Routes (63 has to consume namespace + notebook name)

Reply:
Thanks for the ping @andrewballantyne 
As you rightly pointed, the namespace is used in OpenShift Route creation if use default routing, that use
<route-name>+<namespace> for host creation, and host itself has 63 limit.
https://access.redhat.com/solutions/3638581

It would best, if we can keep the limit lower than 63.

Reply:
Would a maxLength of something lower, perhaps 40 work better? Open to a recommended length if y'all have a specific number in mind.